Update BLorient686.xml

MarcinKrawczuk opened the pull request:
https://github.com/BetaMasaheft/Documentation/issues/2709

eu-genia commented on LondonBritishLibrary/orient/BLorient686.xml:
```diff
@@ -867,6 +867,7 @@ in all supplications, together with those of dowager queen <persName ref="PRS268
          <change who="MKr" when="2021-10-05">Completed description</change>
          <change when="2022-06-15" who="DR">Added layout</change>
          <change when="2023-07-11" who="VP">Added collation; added rubrication; added textual dimensions, with margins; updated outer dimensions; updated binding description, ruling and condition; added cues to the hand description.  The data were collected directly at the British Library, on the days 23 and 24 of Febrary 2023, by Vitagrazia Pisani, within the frame work of the DFG Project 'The G ...
+         <change when="2024-1218"  who="MKr">Corrected work ID in ms_i1.35</change>      
```
![image](https://github.com/user-attachments/assets/782f5179-658c-46ff-83c7-d61e73d19531)

MarcinKrawczuk commented:
Oops, sorry. Corrected.

śr., 18 gru 2024 o 09:09 Eugenia ***@***.***> napisał(a):

> ***@***.**** requested changes on this pull request.
> ------------------------------
>
> In LondonBritishLibrary/orient/BLorient686.xml
> <https://github.com/BetaMasaheft/Manuscripts/pull/2750#discussion_r1889792177>
> :
>
> > @@ -867,6 +867,7 @@ in all supplications, together with those of dowager queen <persName ref="PRS268
>           <change who="MKr" when="2021-10-05">Completed description</change>
>           <change when="2022-06-15" who="DR">Added layout</change>
>           <change when="2023-07-11" who="VP">Added collation; added rubrication; added textual dimensions, with margins; updated outer dimensions; updated binding description, ruling and condition; added cues to the hand description.  The data were collected directly at the British Library, on the days 23 and 24 of Febrary 2023, by Vitagrazia Pisani, within the frame work of the DFG Project 'The Gǝʿǝz Version of the Passio of St Cyricus (Gadla Qirqos): A Critical Edition, Translation, and Commentary', project number: 468455109 (Principal investigator: Vitagrazia Pisani</change>
> +         <change when="2024-1218"  who="MKr">Corrected work ID in ms_i1.35</change>
>
> image.png (view on web)
> <https://github.com/user-attachments/assets/782f5179-658c-46ff-83c7-d61e73d19531>
>
> —
> Reply to this email directly, view it on GitHub
> <https://github.com/BetaMasaheft/Manuscripts/pull/2750#pullrequestreview-2511084702>,
> or unsubscribe
> <https://github.com/notifications/unsubscribe-auth/AO62KJWOKQPYC4VJ6H5CX632GEUVHAVCNFSM6AAAAABTZOZJKGVHI2DSMVQWIX3LMV43YUDVNRWFEZLROVSXG5CSMV3GSZLXHMZDKMJRGA4DINZQGI>
> .
> You are receiving this because you authored the thread.Message ID:
> ***@***.***>
>
